.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is there any special reason why you create this file instead of adding it in the package.json?